### DISCUSSION TITLE
URI for You Tube Playlists

### DISCUSSION THREAD

**Discussion by jdekoven** [music-assistant/support#1071] [Q&A]
My apologies if this has been mentioned but I've searched and don't see anything on this.  In using the mass.queue command for Spotify it seems I can just type in the playlist and it plays, so for instance "John's Playlist", but doesn't seem like that works with You Tube Music.  I am a premium subscriber to both.  I've tried the name, URL etc... but nothing works.  Any suggestions?

**Comment by OzGav** [comment]
Outdated v1 question
